django/core/management/utils.py

```diff
@@ -76,10 +76,10 @@ def find_command(cmd, path=None, pathext=None):
 
 def get_random_secret_key():
     """
-    Return a 50 character random string usable as a SECRET_KEY setting value.
+    Return a 50 character random string (with insecure prefix) usable as a SECRET_KEY setting value.
     """
     chars = 'abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)'
-    return get_random_string(50, chars)
+    return 'insecure-default-' + get_random_string(50, chars)
```

Comment:
I don't think we can make the change here: it's a perennial SO/blog suggestions to import `get_random_secret_key()` to generate new ones. We'll break people's workflows if we mark such keys as insecure. 

The suggestion from the mailing list thread (I thought) was to just adjust the default settings (template) to prefix the starter key there as insecure.